finetune/full.py

```diff
@@ -143,8 +145,34 @@ def train(
     )
 
     validate(fabric, model, val_data, tokenizer, max_iters=2)  # sanity check
+    throughput = ThroughputMonitor(fabric, window_size=5)
+
+    with torch.device("meta"):
+        meta_model = GPT(model.config)
+        x = torch.randint(0, 1, (micro_batch_size, meta_model.config.block_size))
+        model_fwd = lambda: meta_model(x)
+        model_loss = lambda y: chunked_cross_entropy(y, x, chunk_size=0)
+        measured_flops = measure_flops(meta_model, model_fwd, model_loss)
+        fabric.print(f"Measured TFLOPs: {measured_flops * fabric.world_size / 1e12:.2f}")
+        del meta_model, x
```

Comment:
I removed the FLOPs/MFU code from the fine-tuning scripts in https://github.com/Lightning-AI/lit-gpt/pull/688. I don't think these limitations have changed

Reply:
sure, this isn't a must. Can remove it